CHANGELOG.md

```diff
@@ -1,6 +1,101 @@
 Change Log
 ==========
 
+Version 3.1.0 *(2020-01-24)*
+----------------------------
+
+This is a new minor release of Chucker. Please note that this minor release contains multiple new features (see below) as well as multiple bugfixes. 
+
+**Summary of Changes**
+
+* The library is now fully converted to Kotlin!
+* The whole UI has been revamped to support Dark Theme.
+* The library is now using `ViewModel` internally to handle state changes.
```

Comment:
We can't claim that at the moment, since `ViewModel` is used only for `Transaction` screen. Let's postpone this letter to the next release when we (hopefully) have the whole library switched to `ViewModel` approach.

Reply:
> In general looks good, but I would suggest quite different structure, where we have updates, bugfixes, etc. listed in separate sections. For example, I like structure used by [Realm](https://github.com/realm/realm-java/blob/master/CHANGELOG.md) [](/realm/realm-java/blob/HEAD@{2020-01-22T08:32:52Z}/CHANGELOG.md), [Dagger ](https://github.com/google/dagger/releases) or [Material Components](https://github.com/material-components/material-components-android/releases).
> Ready to help here if needed.

I've updated it with sections. Please let me know if you have suggestions.